lib/vagrant-libvirt/templates/private_network.xml.erb

```diff
@@ -2,6 +2,10 @@
   <name><%= @network_name %></name>
   <bridge name="<%= @network_bridge_name %>" />
 
+  <% if @network_domain_name %>
+    <domain name="<%= @network_domain_name %>" />
```

Comment:
I think you want the `localOnly='yes'` option here also.

Reply:
Any idea how libvirt behaves when providing an option that is not supported?
Reason of asking: that option is available since `1.2.12` of libvirt. [Debian](https://packages.debian.org/search?keywords=libvirt0&searchon=names&exact=1&suite=oldstable&section=all) for example is on `1.2.9`.

Reply:
I'm guessing based on libvirt code that it will work just fine on older versions but it might need testing. You can also make `localonly` an option to cover that case.